src/Evolution/Systems/Cce/Actions/InitializeCharacteristicEvolutionVariables.hpp

```diff
@@ -125,18 +125,23 @@ struct InitializeCharacteristicEvolutionVariables {
       const Parallel::GlobalCache<Metavariables>& /*cache*/,
       const ArrayIndex& /*array_index*/, const ActionList /*meta*/,
       const ParallelComponent* const /*meta*/) {
-    const size_t l_max = db::get<Spectral::Swsh::Tags::LMaxBase>(box);
+    initialize_impl(make_not_null(&box));
+    return {Parallel::AlgorithmExecution::Continue, std::nullopt};
+  }
+
+  template <typename TagList>
+  static void initialize_impl(const gsl::not_null<db::DataBox<TagList>*> box) {
```

Comment:
Weird function name (there's no `initialize` for this to be implementing), but since it's not a new choice made in this PR I won't object.